lib/config/config-validator.js

```diff
@@ -123,40 +124,66 @@ function validateEnvironment(environment, source) {
         return;
     }
 
-    if (Array.isArray(environment)) {
-        throw new Error("Environment must not be an array");
-    }
+    Object.keys(environment).forEach(env => {
+        if (!Environments.get(env)) {
+            const message = [
+                source, ":\n",
+                "\tEnvironment key \"", env, "\" is unknown\n"
+            ];
+
+            throw new Error(message.join(""));
+        }
+    });
+}
 
-    if (typeof environment === "object") {
-        Object.keys(environment).forEach(env => {
-            if (!Environments.get(env)) {
-                const message = [
-                    source, ":\n",
-                    "\tEnvironment key \"", env, "\" is unknown\n"
-                ];
-
-                throw new Error(message.join(""));
-            }
-        });
-    } else {
-        throw new Error("Environment must be an object");
+/**
+ * Validates a rules config object
+ * @param {Object} rulesConfig The rules config object to validate.
+ * @param {string} source The location to report with any errors.
+ * @returns {void}
+ */
+function validateRules(rulesConfig, source) {
+    if (!rulesConfig) {
+        return;
     }
+
+    Object.keys(rulesConfig).forEach(id => {
+        validateRuleOptions(id, rulesConfig[id], source);
+    });
 }
 
 /**
- * Validates an entire config object.
+ * Formats an array of schema validation errors.
+ * @param {Array} errors An array of error messages to format.
+ * @returns {string} Formatted error message
+ */
+function formatErrors(errors) {
+    return errors.map(error => `\t- Property "${error.field.replace(/^(data\.)/, "")}" ${error.message}. Current value is: "${JSON.stringify(error.value)}".\n`).join("");
```

Comment:
Yeah, I was going to say the same thing here. We use backticks in a lot of other places; I think that would work pretty well here.